components/landingPage/Mission.tsx

```diff
@@ -0,0 +1,78 @@
+import Image from "next/image";
+import InfoCard from "./InfoCard";
+
+export default function Mission() {
+	/**
+	 * The "Our Mission" Section Of The Landing Page
+	 */
+	return (
+		<div className="normal-bg flex flex-col min-h-screen h-full" id="mission">
+			<h2 data-testid="mission-title" className="section-header">
```

Comment:
i don't know this was intentional but padding-bottom is give 1.25rem but padding-top was none and this looks a bit asymmetrical, i don't know the tailwind syntax but tried to give padding-top: 1.25rem on ff devtool for balancing it out just as other header and it looks symmetrical. just let u know!

Reply:
It was intentional and part of the section-header class. Anyway making it symmetrical is not an hard task, if you think that is an improvement I can modify the style.

Reply:
Ah i see, no don't bother ~_~ i don't have any problem with that. i just wanted to ask